### PR TITLE
Physicslist

### PR DESCRIPTION
**Pull request by yramachers**
This PR implements a custom made physics list, copied mostly from the EMStandard_option4 high precision low energy list but exchanged electron scattering from multiple- to single scattering model, eDPWACoulombScatteringModel.
Note, the Geant4 10.7 eDPWACoulombScatteringModel implementation has an error in lines 89/90 which are removed in the patch as implemented here (commented). 
Single scattering is a requirement for our application since our target is essentially a high vacuum with at most 10^14 cm^-3 H-atoms. The low-energy ionization is still a Geant4 continuous energy loss (dE/dx) model which is supposed to be precise (from data) but not really what we will need eventually (additional free electron after ionization). Nevertheless, passing electrons in our target appear to show continuous energy loss to some degree, so that model seems to work fine. Just lacking those ionization e-.

### CODE REVIEW

File: examples/ScatteringExample/src/G4patcheDPWACoulombScatteringModel.cc [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+G4patcheDPWACoulombScatteringModel::G4patcheDPWACoulombScatteringModel(G4bool ismixed, G4bool isscpcor, G4double mumin)
+: G4VEmModel("eDPWACoulombScattering"),
+  fIsMixedModel(ismixed),
+  fIsScpCorrection(isscpcor),
+  fMuMin(mumin),
+  fTheDCS(nullptr),
+  fParticleChange(nullptr)
+{
+  SetLowEnergyLimit (  0.0*CLHEP::eV);  // ekin = 10 eV   is used if (E< 10  eV)
```
Comment (TomGoffrey) [inline examples/ScatteringExample/src/G4patcheDPWACoulombScatteringModel.cc]: I realise this is a copy-paste from the original source code, but is this comment correct? (Comment implies lower limit of 10eV, code suggests 0?)

File: examples/ScatteringExample/src/QTNMPhysicsList.cc [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+  param->SetLowestElectronEnergy(100*CLHEP::eV);
+  param->SetNumberOfBinsPerDecade(20);
+  param->ActivateAngularGeneratorForIonisation(true);
+  param->SetStepFunction(0.2, 10*CLHEP::um);
+  param->SetStepFunctionMuHad(0.1, 50*CLHEP::um);
+  param->SetStepFunctionLightIons(0.1, 20*CLHEP::um);
+  param->SetStepFunctionIons(0.1, 1*CLHEP::um);
+  param->SetMuHadLateralDisplacement(true);
+  param->SetFluo(true);
+  param->SetMscThetaLimit(0.0);
```
Comment (TomGoffrey) [inline examples/ScatteringExample/src/QTNMPhysicsList.cc:75]: Shouldn't this result in `PolarAngleLimit()` returning zero, which in turn would give `fMuMin = 0` in (the now commented) line 89 of G4eDPWACoulombScatteringModel, which (judging from other comments in the source code) corresponds to the single scattering case?

File: examples/ScatteringExample/src/G4patcheDPWACoulombScatteringModel.cc [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+}
+
+
+void G4patcheDPWACoulombScatteringModel::Initialise(const G4ParticleDefinition* pdef,
+                                               const G4DataVector& prodcuts)
+{
+  if(!fParticleChange) {
+    fParticleChange = GetParticleChangeForGamma();
+  }
+  // fMuMin        = 0.5*(1.0-std::cos(PolarAngleLimit()));
```
Comment (TomGoffrey) [inline examples/ScatteringExample/src/G4patcheDPWACoulombScatteringModel.cc]: Is it worth raising this with the Geant4 devs? Is there an active bug report? (I looked, but couldn't find one)

**Review by TomGoffrey — COMMENTED** [review]
I think this is OK as is, but I've raised a few places where I think some clarification would be a good idea.

In particular I'm a bit confused by `param->SetMscThetaLimit(0.0);` not resulting in fMuMin = 0.0, which I think is the intention, but I also lack the knowledge of Geant4 to properly follow the code.

File: examples/ScatteringExample/src/QTNMPhysicsList.cc (hunk 2 above)
Comment (yramachers) [reply examples/ScatteringExample/src/QTNMPhysicsList.cc:75]: Correct, I agree. Didn't see that this happens in the Coulomb process code.

File: examples/ScatteringExample/src/G4patcheDPWACoulombScatteringModel.cc (hunk 3 above)
Comment (yramachers) [reply examples/ScatteringExample/src/G4patcheDPWACoulombScatteringModel.cc]: patched version removed. Still think it is wrong to overwrite input parameter but at least it doesn't matter.

**Review by TomGoffrey — APPROVED** [review]

**Pull request merged**